python/cudf/cudf/core/dataframe.py

```diff
@@ -5499,14 +5510,16 @@ def isin(self, values):
     #
     # Stats
     #
-    def _prepare_for_rowwise_op(self):
+    def _prepare_for_rowwise_op(self, method, skipna):
         """Prepare a DataFrame for CuPy-based row-wise operations.
         """
 
-        if any([col.nullable for col in self._columns]):
+        if method not in _cupy_nan_methods_map and any(
+            [col.nullable for col in self._columns]
+        ):
             msg = (
-                "Row-wise operations do not currently support columns with "
-                "null values. Consider removing them with .dropna() "
+                "Row-wise operations do not currently support columns "
+                "with null values. Consider removing them with .dropna() "
                 "or using .fillna()."
```

Comment:
This error string should be updated to indicate that the requested op isn't supported instead of row-wise ops in general

Reply:
Done ✅